src/Http/Controllers/Admin/XlsformTemplateCrudController.php

```diff
@@ -58,6 +60,10 @@ protected function setupListOperation(): void
             'csv_name' => 'CSV File Name',
         ]);
         CRUD::column('available')->type('boolean')->label('Is the form available for live use?');
+
+        // add select ODK variables button
+        // P.S. Uncomment this line will lead to error
+        // $this->crud->addButtonFromView('line', 'select', 'select', 'end');
     }
```

Comment:
This doesn't work because it is looking inside the default namespace for Laravel Backpack buttons, rather than the OdkLink Package's views. 

Normally, to reference a view inside a package, you need to specify the package's namespace, e.g: 

- `view('odk-link::buttons.select')`

However, the following doesn't work either: 

- `$this->crud->addButtonFromView('line', 'select', odk-link::buttons.select', 'end');`

Digging into the `addButtonFromView()` method tells us why - that method prepends `crud::buttons` to the view name, which means we can't use this for something inside a different package. 

Instead, I would either go to the more generic function `$this->addButton()`, or use the `fluent` syntax - there's an example of that in the SubmissionCrudController: 

```
        Crud::button('reprocess')
            ->stack('line')
            ->view('odk-link::buttons.submissions.reprocess');
```

Reply:
Dave, Thanks a lot for your advice, it is now working.